test/e2e/loopback_test.go

```diff
@@ -206,40 +206,40 @@ var _ = ginkgo.Describe("Loopback registration [development]", func() {
 		}
 
 		var (
-			spoke         *clusterv1.SpokeCluster
-			spokeClusters = clusterClient.ClusterV1().SpokeClusters()
+			managedCluster  *clusterv1.ManagedCluster
+			managedClusters = clusterClient.ClusterV1().ManagedClusters()
 		)
 
 		ginkgo.By(fmt.Sprintf("Waiting for SpokeCluster %q to exist", clusterName))
 		err = retry.OnError(retry.DefaultRetry, errors.IsNotFound, func() error {
 			var err error
-			spoke, err = spokeClusters.Get(context.TODO(), clusterName, metav1.GetOptions{})
+			managedCluster, err = managedClusters.Get(context.TODO(), clusterName, metav1.GetOptions{})
 			return err
 		})
 		gomega.Expect(err).ToNot(gomega.HaveOccurred())
-		gomega.Expect(spoke.Spec.HubAcceptsClient).To(gomega.Equal(false))
+		gomega.Expect(managedCluster.Spec.HubAcceptsClient).To(gomega.Equal(false))
 
 		ginkgo.By(fmt.Sprintf("Accepting SpokeCluster %q", clusterName))
 		err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			var err error
-			spoke, err = spokeClusters.Get(context.TODO(), spoke.Name, metav1.GetOptions{})
+			managedCluster, err = managedClusters.Get(context.TODO(), managedCluster.Name, metav1.GetOptions{})
 			gomega.Expect(err).ToNot(gomega.HaveOccurred())
 
-			spoke.Spec.HubAcceptsClient = true
-			spoke, err = spokeClusters.Update(context.TODO(), spoke, metav1.UpdateOptions{})
+			managedCluster.Spec.HubAcceptsClient = true
+			managedCluster, err = managedClusters.Update(context.TODO(), managedCluster, metav1.UpdateOptions{})
 			return err
 		})
 		gomega.Expect(err).ToNot(gomega.HaveOccurred())
 
 		ginkgo.By("Waiting for SpokeCluster to have HubAccepted=true")
 		err = wait.Poll(1*time.Second, 30*time.Second, func() (bool, error) {
 			var err error
-			spoke, err := spokeClusters.Get(context.TODO(), clusterName, metav1.GetOptions{})
+			managedCluster, err := managedClusters.Get(context.TODO(), clusterName, metav1.GetOptions{})
 			if err != nil {
 				return false, err
 			}
 
-			condition := helpers.FindSpokeClusterCondition(spoke.Status.Conditions, "HubAcceptedSpoke")
+			condition := helpers.FindManagedClusterCondition(managedCluster.Status.Conditions, "HubAcceptedSpoke")
```

Comment:
we may need to change the the condition name

Reply:
done